core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -221,9 +221,9 @@ export class ContributorDashboardPageComponent
 
     this.activeTabName = 'myContributionTab';
 
-    // TODO(#13015): Remove use of unknown as a type.
     this.tabsDetails = {
       ...ContributorDashboardConstants.CONTRIBUTOR_DASHBOARD_TABS_DETAILS
+    // TODO(#13015): Remove use of unknown as a type.
```

Comment:
Hi, for the GSoC milestone 2, you need to remove all the uses of unknown from the code base and resolve this issue: https://github.com/oppia/oppia/issues/13015. If there are any usages of unknown that are valid, explain why they are valid in a comment above the usage. Make sure that your lint check accommodates/ looks for that explanation.

Reply:
[explain](https://github.com/oppia/oppia/pull/16597#discussion_r1032822553).

Add this comment is just make sure, this address in later.

Reply:
Hi @JeeveshGarg -- discussed with @srijanreddy98 and @vojtechjelinek: 

- For the legitimate instances of unknown left in the codebase, there has to be a clear explanation of what is required for it to be removed. E.g. "Remove use of this unknown once jQuery is removed."
    - Legitimate instances of unknown are restricted to these three categories: (a) dependencies, (b) try/except, (c) inherited classes like ExplorationPropertyService.

- In #13015, there should be only these three categories and all other checkboxes should be fully fixed in this PR.

Reply:
Removed all TODO,

Will add proper comment on each and every unknown.

Then add that list in #13015 (later, because need to change des. as well).